app/helpers/application_helper/button/middleware_server_action.rb

```diff
@@ -1,7 +1,6 @@
 class ApplicationHelper::Button::MiddlewareServerAction < ApplicationHelper::Button::Basic
-  needs :@record
 
   def visible?
-    @record.try(:product) != 'Hawkular' && @record.try(:middleware_server).try(:product) != 'Hawkular'
```

Comment:
I guess that in the future we might want to do something else to identify if an action is enabled on certain products (Hawkular, JDG, etc)